src/lib.rs

```diff
@@ -1669,4 +1783,19 @@ mod tests {
         let c3 = Color::from_rgb(143, 111, 76);
         assert_eq!("cmyk(0, 22, 47, 44)", c3.to_cmyk_string(Format::Spaces));
     }
+
+    #[test]
+    fn alpha_roundtrip_hex_to_decimal() {
+        // We use a max of 3 decimal places when displaying RGB floating point
+        // alpha values. This test insures that is sufficient to "roundtrip"
+        // from hex (0 < n < 255) to float (0 < n < 1) and back again,
+        // e.g. hex `80` is float `0.502`, which parses to hex `80`, and so on.
+        for alpha_int in 0..255 {
+            let hex_string = format!("#000000{:02x}", alpha_int);
+            let parsed_from_hex = hex_string.parse::<Color>().unwrap();
+            let rgba_string = parsed_from_hex.to_rgb_float_string(Format::Spaces);
+            let parsed_from_rgba = rgba_string.parse::<Color>().unwrap();
+            assert_eq!(hex_string, parsed_from_rgba.to_rgb_hex_string(true));
+        }
```

Comment:
Alright, I think this should provide proper coverage. This asserts that:

- for every hex alpha value, 0 < n < 255 (`00` - `fe`)
- parsing a color with this alpha, in hex notation
- outputting that color as `rgba()`, with alpha as decimal
- parsing _that_ color
- outputting that color using hex notation
- generates the same value as the original hex input string

For example, `#00000080` parses to a color that stringifies to `rgba(0, 0, 0, 0.502)`, that in turn parses to a color that stringifies to `#00000080`.

I also threw in an `impl FromStr for Color` that calls `parser::parse_color`, that's not required, just thought it may be useful.

Reply:
Awesome, thank you!